src/platform/site-wide/sass/modules/_m-cta-widget.scss

```diff
@@ -4,7 +4,7 @@
     border-left: none;
 
     .fa-angle-down {
-      color: #323A45;
+      color: $color-gray-dark;
```

Comment:
https://github.com/department-of-veterans-affairs/veteran-facing-services-tools/blob/52f75afbcd5e4217be513729e50a0fcc0dc92cf2/packages/formation/sass/base/_b-variables.scss#L73